compose/ui/ui-test/src/skikoMain/kotlin/androidx/compose/ui/test/ComposeUiTest.skiko.kt

```diff
@@ -301,11 +301,13 @@ open class SkikoComposeUiTest @InternalTestApi constructor(
         )
     }
 
+    protected open val platformContext: PlatformContext get() = TestContext()
```

Comment:
Note: It's public APIs for users. I'm not sure that we really want to expose it (even as experimental)

cc @m-sasha as owner of this API